aries-site/src/examples/templates/forms/do&dont/index.js

```diff
@@ -0,0 +1 @@
+export * from './ColumnFormDont';
```

Comment:
```suggestion
export * from './ColumnFormDont';

```
need form inline

Reply:
also, the do&dont as a file name feels a little weird to have "&". Can we name it `do-dont`?

Reply:
yeah thats fine